spectroscopy/spectra_generator.md

```diff
@@ -99,15 +93,16 @@ Andreas Faisst, Jessica Krick, Shoubaneh Hemmati, Troy Raen, Brigitta Sipőcz, D
 
 This cell will install them if needed:
 
-```{code-cell}
+```{code-cell} ipython3
 # Uncomment the next line to install dependencies if needed.
-# !pip install -r requirements_spectra_generator.txt
+!pip install -r requirements_spectra_generator.txt
```

Comment:
Its hard for me to remember this because I need it uncommented to run, and then forget to go back and comment it out.  Is there some easy way to have both work, ie some if statement around it, or???

Reply:
Ahh, interesting. I'll open a follow-up issue for this over at the image repo as I think it would be a nicer user experience to have persistency. (But also, installing all these requirements file is on the request list)

Reply:
>  ie some if statement around it, or

While we did some conditionals and then install steps in the early version, I don't think it's particularly good to ship demo notebooks to end users where installations are hidden away that way. cc @troyraen 

As a workflow workaround for you as a notebook developer rather than end user could be to pip install everything from a terminal session freshly after a new login and then propagate the list of missing dependencies as an issue in the fornax-image repo, so we can include it in the kernel.